src/components/result-link.tsx

```diff
@@ -0,0 +1,14 @@
+import { Result } from "@coveo/headless";
+import { FunctionComponent } from "react";
+
+interface LinkProps {
+  result: Result;
+}
+
+export const ResultLink: FunctionComponent<LinkProps> = (props) => {
+  return (
+    <a href={props.result.clickUri} style={{color: '#00ADFF'}}>
```

Comment:
In general, shouldn't we do style in the stylesheet?

Reply:
Usually I fully agree. In this case, I thought it made more sense to put the style directly in the component so that users are aware that it's something we've added and not part of Headless.
That being said, we can assume our users know how styling works, and so maybe realizing that _no_ styling is included in Headless is on them. What do you think?

Reply:
In general, I think if we should try to use good programming practices in our examples, unless it really complexifies stuff.

In this case, if we add a class to the element for styling purposes, they'll probably make the connection quickly themselves. So yes, I would personally assume our users know how styling works :)